gulpfile.js

```diff
@@ -236,6 +239,24 @@ gulp.task(
   )
 )
 
+gulp.task(
+  'copy:govuk_frontend_assets:fonts',
+  copyFactory(
+    'fonts from the GOVUK frontend assets',
+    govukFrontendFontsFolder,
+    assetsFolder + '/fonts'
```

Comment:
This should actually be `staticFolder`; see https://github.com/alphagov/digitalmarketplace-user-frontend/pull/176

Reply:
ah ok